src/library/scala/concurrent/duration/Duration.scala

```diff
@@ -705,7 +705,7 @@ final class FiniteDuration(val length: Long, val unit: TimeUnit) extends Duratio
 
   final def isFinite() = true
 
-  final def toCoarsest: Duration = {
+  final def toCoarsest: FiniteDuration = {
```

Comment:
True, always good for course compatibility, in case the super class makes that method concrete.